tools/cldr-apps/js/src/views/VettingSummary.vue

```diff
@@ -41,7 +41,7 @@
               title="Show the indicated snapshot of the Priority Items Summary"
```

Comment:
^ the title here could contain the snapshotId

Reply:
good idea